chalicelib_fourfront/checks/helpers/wfrset_utils.py

```diff
@@ -195,9 +195,9 @@ def step_settings(step_name, my_organism, attribution, overwrite=None):
     },
     {
         "app_name": "encode-chipseq-aln-chip",
-        "workflow_uuid": "4dn-dcic-lab:wf-encode-chipseq-aln-chip",
+        "workflow_uuid": "212a9c91-25d6-473f-b56b-8dd93958c580",
```

Comment:
despite the property name, might it be better to use the alias as was formerly done in case the workflow is added to a new environment (with a different uuid being generated) - perhaps unlikely but if at all possible?

Reply:
Yes, I was aiming to make it consistent with other workflows, since I made the new UUIDs are consistent (for the current envs). The aliases also are the same for both new and old versions, which maybe also was a factor? It needs to know the new and old version are the same workflow, but only run the new one.

Reply:
Oops, I see your confusion--the aliases are definitely not the same and the app name keeps the versions together. I do think it is more consistent with other workflows to use the uuid. It is also hard to query aliases for someone going from this workflow setting to the workflow on the portal, right?